packages/rich-text-editor/src/RichTextEditor/RichTextEditor.tsx

```diff
@@ -82,6 +82,15 @@ export const RichTextEditor = (props: RichTextEditorProps): JSX.Element => {
   )
   const [labelId] = useState<string>(labelledBy || v4())
   const [editorId] = useState<string>(v4())
+  const validationMessageAria = validationMessage
+    ? `${editorId}-rte-validation-message`
+    : ""
+  const descriptionAria = description ? `${editorId}-rte-description` : ""
+  const ariaDescribedBy = classnames(
```

Comment:
Was a little confused by `classnames` here since we so often just use it for styles, but given it should just concat the ids I suppose that should work just fine

Reply:
Based on the docs for it, falsey values are ignore so even better

Reply:
Geoff said the same thing but ya that's all this function does, it should really be called `joinArrayOfStringsWithSpacesIgnoringFalsy()`, so much more catchy 😂 

Maybe in the KAIO version I'll use `join()` given two people have said this now.